micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -0,0 +1,144 @@
+
+## [1.5.32]
+- Added newer CSS DIGITv2 and corrected a few existing issues.
+
+## [1.5.31]
+- Corrected the CSS for inbox composers and default core UI.
+
+## [1.5.30]
+- Updated the CSS for dynamic dropdown filter DSS.
+
+## [1.5.29]
+- Fixed layout issues.
```

Markdownlint

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)